module-2/backend/week_two.md

```diff
@@ -25,14 +32,24 @@ end
 ```
 
 Now how would you find the owner of the team with an id of 4?
+Owner name: Team.find(4).owner.name
 
 3. What do they allow you to do?
+???
 7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+Assuming there is only one teacher a teacher has many students
 8. Define foreign key, primary key, and schema.
+Foreign Key: teacher_id (student table)
+Primary Key: id (teacher table)
+Teacher Table: id, name
+Student Table: id, name, teacher_id
+
 9. Describe the relationship between a foreign key on one table and a primary key on another table.
+This is the key that connects two tables in a one to many relationship. The primary key is on the one side and the foreign key is on the many side.
 10. What are the parts of an HTTP response?
+Verb, Path, Protocol, Parameters
```

Comment:
For a response we have headers and a body. The first of the headers includes a status (e.g. 200, 404, etc.)